.github/ISSUE_TEMPLATE/Currency_update_request.yml

```diff
@@ -0,0 +1,35 @@
+name: Currency Update Request
+title: "[Currency_Update]: "
+description: "Update request to add support for specific version on ppc64le for available currency package Or fix currency "
+labels: "update currency"
+projects: "ppc64le/1"
+assignees: "ICH"
+body:
+  - type: markdown
+    attributes:
+      value: >
+        Github is reserved for bug reports and feature requests; it is
+        not the place for general questions. 
```

Comment:
Which brings up an interesting question - where should users raise questions?  And is there any real reason why they couldn't go in here?  Or should general questions go through one of the other templates?  Maybe just be definitive about what *is* allowed:  "Please complete the following fields to request an addition to the currency portal, a fix to an existing package, or an update to an existing package".

Reply:
updated to be like `Please complete the following fields to request a fix to an existing package, or an update to an existing package.`

Reply:
I think if question is about update in currency of package or related , it can be accepted here.
Removed the ambiguity from description